lib/components/List/Grid/index.jsx

```diff
@@ -58,17 +64,23 @@ const renderListColumns = ({ keyMap, item, liIndex }) => {
 };
 
 const Grid = ({
+  id,
   classes,
   items,
   keyMap,
   withLoader,
 }) => (
   items.length > 0 || !withLoader ?
-    <ul className={`ui-grid list-group ${classes}`}>
+    <ul id={id} className={`ui-grid list-group ${classes}`}>
```

Comment:
Is it fine to have an empty string as id, i.e. no id, or is this going to cause errors?

Reply:
Have you tried something like this?

id={id && `${id}-${column.key}`}